norona_marynicole/countries.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="countries.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Countries of the World</h1>
+      <input
+        type="text"
+        id="countryInput"
+        placeholder="Enter country name..."
+      />
```

Comment:
Remove unnecessary breaking of lines

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2mb5Qre33CFfRuTtpvAN1dTOpbICXFQJSlA9Pa6x8KcjHwHelDlK8iGzQ#slide=id.g29e6ce31160_0_27